src/App.js

```diff
@@ -1,25 +1,115 @@
-import React from 'react';
+import React, {useEffect, useRef} from 'react';
+import { Link } from 'react-scroll';
 import './App.css';
 
 function App() {
```

Comment:
please move the navbar to navbar component file and separate navbar and welcome as it is in the file structure.